app/models/user.rb

```diff
@@ -97,6 +97,17 @@ def self.cleanup_unverified_users
     by_unverified.older_than_one_year.destroy_all
   end
 
+  def sheer_supported?
+    {
+      '1'   => 'United States & Canada',
+      '27'  => 'South Africa',
+      '44'  => 'United Kingdom',
+      '61'  => 'Australia',
+      '64'  => 'New Zealand',
+      '353' => 'Ireland',
+    }.key?(country_code&.strip)
+  end
+
```

Comment:
Oh, however, I think it would be better to store the hash as `COUNTRY_CODES` in the `User` and then this method would just do: `COUNTRY_CODES.keys.include? country_code&.strip` 👌

Reply:
Even better, no need to `strip` the country code here. We can add it to https://github.com/openstax/accounts/blob/master/app/models/user.rb#L334. So this will be `COUNTRY_CODES.keys.include? country_code`

Reply:
I thought of putting the codes as a static...but thought to keep the country codes and the check close together, for easy changing.  can change if you want....

Reply:
thought of putting these as a constant..but thought it looked nice here as once cohesive unit.  can move to constant if you wish...

Reply:
True. That works.